File: C Sharp/allComments.cs
using System;
using System.Net;
using System.Text;
using System.IO;

class AllComments
{
	public static void Main(string[] args)
	{
		string auth_token = "59550a0e2b1a864a31bef962363e029f"; // YOUR_AUTH_TOKEN
		string org_id = "652853630"; //YOUR_ORG_ID

		string ticket_id = "215666000000074114";

		if (ticket_id != "")
		{

			HttpWebRequest request = (HttpWebRequest)WebRequest.Create("https://desk.zoho.com/api/v1/tickets/" + ticket_id + "/comments");
			request.ContentType = "application/json";
			request.Method = "GET";
			request.Headers["Authorization"] = auth_token;
			request.Headers["orgId"] = org_id;

			try
			{
				HttpWebResponse response = (HttpWebResponse)request.GetResponse();
				Stream receiveStream = response.GetResponseStream();
				StreamReader readStream = new StreamReader(receiveStream, Encoding.UTF8);
				Console.WriteLine("Response Code : " + (int)response.StatusCode);

				if ((int)response.StatusCode == 200)
				{
					Console.WriteLine(readStream.ReadToEnd());
                }
                else{
                    if ((int)response.StatusCode == 204) Console.WriteLine("{}");
				}
				response.Close();
				readStream.Close();

			}
			catch (WebException e)
			{
				Console.WriteLine("Request failed, Response Code : " + (int)((HttpWebResponse)e.Response).StatusCode + " \nResponse Message : ");
				var message = new StreamReader(e.Response.GetResponseStream()).ReadToEnd();
				Console.WriteLine(message);
			}

		}
		else
		{
			Console.WriteLine("ERROR : Ticket ID is missing");
		}

	}
}

File: PHP/allComments.php
<?php

    $auth_token = '59550a0e2b1a864a31bef962363e029f'; //your_auth_token
    $org_id=652853630; //your_organization_id
    $ticket_id="215666000000074114"; //Ticket ID

    $headers=array(
            "Authorization: $auth_token",
            "orgId: $org_id",
            "contentType: application/json; charset=utf-8",
    );

    $params="limit=15&sortBy=commentedTime"; //options as parameters

    if($ticket_id){
        $url="https://desk.zoho.com/api/v1/tickets/$ticket_id/comments?$params";

        $ch= curl_init($url);
        curl_setopt($ch,CURLOPT_HTTPHEADER,$headers);
        curl_setopt($ch,CURLOPT_RETURNTRANSFER,TRUE);
        curl_setopt($ch,CURLOPT_HTTPGET,TRUE);

        $response= curl_exec($ch);
        $info= curl_getinfo($ch);

        if($info['http_code']==200){
            echo "<h2>Request Successful, Response:</h2> <br>";
            echo $response;
        }
        else{
            echo "Request not successful. Response code : ".$info['http_code']." <br>";
            echo "Response : $response";
        }

        curl_close($ch);
    }
    else{
        echo "ERROR : Ticket ID is missing";
    }

?>
